frontend/packages/console-dynamic-plugin-sdk/src/app/redux-types.ts

```diff
@@ -16,6 +16,7 @@ export type ImpersonateKind = {
 
 export type CoreState = {
   user?: UserInfo;
+  userResource?: any; // UserKind from k8s API
```

Comment:
this should be `UserKind` from '@console/internal/module/k8s/types';

```suggestion
  userResource?: UserKind;
```